packages/divination-oracles/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "divination-oracles"
+title: "Divination Oracles"
+description: A package that simulates various divination oracles 
+version: 0.1.0
+author: Olivia Montoya
+tags: ["fun", "divination", "gaming"]
```

Comment:
May I suggest adding "cards" and "tarot" tags to make it easier to find in Espanso Hub?

Reply:
Will add that